static/js/home.js

```diff
@@ -35,7 +36,7 @@ function getEvents(){
             'right':ne.lng,
             'top':ne.lat,
             'down':sw.lat,
-            'time_of_day_range': [0, 6*60*60]
+            'facetObj':facetObj
```

Comment:
Before sending to the backend, Need to make the parameter in consistency with the documentation. 

https://docs.google.com/document/d/1mnAF3Tm8j--zzLt3CNVAykLeUJn-9H_pc693UG7MjY8/edit